modules/ding_webtrekk/ding_webtrekk.module

```diff
@@ -62,10 +62,10 @@ function ding_webtrekk_page_alter(&$page) {
  * Track ting search results.
  */
 function ding_webtrekk_panels_pane_content_alter($content, $pane, $args, $contexts) {
-  if ($pane->type == 'search_result' && $pane->configuration['type'] == 'ting_search') {
+  if ($pane->type == 'search_result' && $pane->configuration['type'] == 'ting_search' && strpos(current_path(), 'search/ting') === 0) {
```

Comment:
The combination of the implemented hook and the dependency on the current path does not seem right. The reliance on panes seems to be unnecessary noise.

How about using another hook instead and just checking whether `drupal_static('ting_search_results', FALSE)` is set? Perhaps `hook_page_alter` is really the way to go. Another approach could be `hook_search_page`

Reply:
I don't know, this was the way it was done in the old module. Just wanted to ensure that it only gets added on the correct page.

Reply:
If I remember correctly it was done this way because of:

Reply:
The static variable drupal_static('ting_search_results', FALSE) is set in ting_search_search_execute. So maybe it's safe to check for it in hook_page_alter. 

Should I make this update and remove hook_panels_pane_content_alter() all together?

Just have the feeling that I forgotten something and it was made this way for a reason (made it myself, but can't remember tbh)

Reply:
Looked into hook_search_page. It seems it's only invoked on the module implementing the search type?

Reply:
@holt83: Then I think we should switch to `hook_page_alter` and check current path. `hook_panels_pane_content_alter()` can be removed entirely.

Reply:
Ok. I will update PR with this approach